internal/comby/types.go

```diff
@@ -0,0 +1,37 @@
+package comby
+
+type Input interface {
+	Value() string
+}
+
+type ZipPath struct {
```

Comment:
Why not do this?

```suggestion
type ZipPath string
```

Then, instead of `ZipPath{value: "foobar"}`, you simply do `ZipPath("foobar")`

Reply:
Hmmm, but how can I case out on `ZipPath` versus `DirPath` then? Related comment: https://github.com/sourcegraph/sourcegraph/pull/6244/files#r339956808